.github/workflows/example-project.yml

```diff
@@ -35,13 +35,25 @@ jobs:
         run: |
           cargo install --path .
 
+      - name: Set MSVC/dumpbin path
+        if: matrix.toolchain-suffix == '-msvc'
+        run: |
+          $MsvcPath = vswhere -latest -products * -find "VC\Tools\MSVC\*\bin\Hostx64\x64" |
+                      Select-Object -Last 1
+          echo "::add-path::$MsvcPath"
+
       - name: Build example project
         working-directory: example-project
         run: |
           cargo cbuild --verbose --release
 
-      - name: Install example project
+      - name: Install into temporary location
+        working-directory: example-project
+        run: |
+          cargo cinstall --verbose --release --destdir=temp
+
+      - name: Copy installed files to /usr/local
+        if: startsWith(matrix.os, 'macos') || startsWith(matrix.os, 'ubuntu')
```

Comment:
I would replace it with: `if: matrix.os == 'macos-latest' || matrix.os == 'ubuntu-latest'`

Reply:
OK, I can do that, but why is that better?

What if `macos` or `ubuntu` versions are added/changed later?

Reply:
Because we want to run the tests only on the last `ubuntu` and `macos` versions, not even the other ones.

Reply:
What about this one instead?

Reply:
I still don't quite understand why you want that change.

My version works always, your version breaks when other OS versions are selected.

And even if we are totally sure that we will never ever change the OS versions, I still don't see any disadvantage in my version.

More importantly, this is supposed to become something like a template for other people, who probably want to use different OS versions.

Reply:
If you want to test this CI script on other operating systems, practically adding more configurations in the matrix, you're perfectly right, but since there are only the `latest` versions for now in the matrix, I thought we could have done a simple comparisons instead of handling strings using the `startsWith` function. 

But now you told me you are interested in adding (in a future) further configurations, so I'm fine with your change :)